R/derive_var_ontrtfl.R

```diff
@@ -186,7 +186,17 @@ derive_var_ontrtfl <- function(dataset,
                                ref_end_window = 0,
                                ignore_time_for_ref_end_date = TRUE,
                                filter_pre_timepoint = NULL,
-                               span_period = NULL) {
+                               span_period = FALSE) {
+  if (is.null(span_period) || span_period %in% c("Y", "y")) {
+    # replace span_period with lgl version
+    span_period <- !is.null(span_period)
+    lifecycle::deprecate_warn(
```

Comment:
```suggestion
   deprecate_warn(
```
For the most part, we have gotten rid of using `::` inside function bodies